sysutils/ntpsec/Portfile

```diff
@@ -19,12 +27,12 @@ homepage            https://www.ntpsec.org/
 conflicts           ntp openntpd
 
 master_sites        ftp://ftp.ntpsec.org/pub/releases/
```

Comment:
Maybe we should transition over to using HTTPS for this mirror? ( https://ftp.ntpsec.org/pub/releases/ )